pepper-apis/src/main/java/org/broadinstitute/ddp/model/kit/KitConfiguration.java

```diff
@@ -92,8 +92,10 @@ public boolean evaluate(Handle handle, String userGuid) {
         }
         if (success && zipCodeSuccess.size() > 0) {
             success = zipCodeSuccess.contains(true);
-            LOG.warn("Study has kit configuration with zip code rule but user's zip code does not"
-                    + " match any accepted zip codes, studyGuid={} userGuid={}", studyGuid, userGuid);
+            if (!success) {
```

Comment:
This is a bug. It's supposed to be wrapped in a `if` block, otherwise we're almost always printing this message.